lms/djangoapps/instructor_task/tasks.py

```diff
@@ -75,11 +75,11 @@ def rescore_problem(entry_id, xmodule_instance_args):
     action_name = ugettext_noop('rescored')
     update_fcn = partial(rescore_problem_module_state, xmodule_instance_args)
 
-    def filter_fcn(modules_to_update):
-        """Filter that matches problems which are marked as being done"""
-        return modules_to_update.filter(state__contains='"done": true')
+    # def filter_fcn(modules_to_update):
+    #     """Filter that matches problems which are marked as being done"""
+    #     return modules_to_update.filter(state__contains='"done": true')
```

Comment:
Commented code should be deleted, but also, is deleting this going to make CAPA rescorable even before a student has submitted scores?